packages/fx-core/src/plugins/resource/bot/v2/index.ts

```diff
@@ -131,6 +140,23 @@ export class BotPluginV2 implements ResourcePlugin {
     return await deployAdapter(ctx, inputs, envInfo, tokenProvider, this.plugin);
   }
 
+  async getQuestionsForUserTask(
+    ctx: v2.Context,
+    inputs: Inputs,
+    func: Func,
+    envInfo: v2.DeepReadonly<v2.EnvInfoV2>,
+    tokenProvider: TokenProvider
+  ): Promise<Result<QTreeNode | undefined, FxError>> {
```

Comment:
Nit: since there's a FxResult alias in result.ts, use it as return type? align with other lifecycle methods.